cjs/index.js

```diff
@@ -8,13 +8,12 @@ module.exports = ({types: t}) => ({
       const {node: {name, value}, parent} = path;
 
       if (
-        t.isJSXNamespacedName(name) ||
         t.isJSXExpressionContainer(value) ||
```

Comment:
If someone uses other plugins that work modifying the JSX elements, these AST nodes could be requeued and revisited.

I suggest returning here if `name.name[0] === "\x01"`.

Reply:
Good hint, thank you! Do you think I should also keep the `t.isJSXNamespacedName(name)` check around?

Reply:
I've answered myself .. I think I should, otherwise I'll mess up with other plugins. Done.